### PR TITLE
Added BAD_VALUE mSensors.size() is zero

### PR DESCRIPTION
**Pull request by telishix**
For celadon added BAD_VALUE mSensors.size() is zero Fix for VtsHalSensorsV2_0TargetTest

Tracked-On: OAM-105025
Signed-off-by: Teli,ShivaniX <shivanix.teli@intel.com>

### CODE REVIEW

File: sensors/2.0/iiohal_mediation_v2.0/Sensors.h [hunk 1]
```diff
@@ -98,6 +98,9 @@ struct Sensors : public ISensorsInterface, public ISensorsEventCallback {
     }
 
     Return<Result> setOperationMode(OperationMode mode) override {
+	if(mode == OperationMode::DATA_INJECTION && !(mSensors.size() > 0)) {
```
Comment (rajucm2007) [inline sensors/2.0/iiohal_mediation_v2.0/Sensors.h:101]: mode == OperationMode::DATA_INJECTION  -> Please surround with parenthesis

Comment (telishix) [reply sensors/2.0/iiohal_mediation_v2.0/Sensors.h]: Done

**Review by rajucm2007 — APPROVED** [review]
LGTM

**Review by rajucm2007 — APPROVED** [review]
LKGM -> approved.
Let us merge this & plan for improvement due to following reason.
size operator return type is unsigned. cannot return negative value...  
possible return values 0 & +ve numbers. Checking against -ve case never be the case.
if(!mSensors.size()))   ---> makes more meaningful

**Comment by vdanix** [comment]
@JeevakaPrabu  could you please reveiw the pr

**Review by vilasrk — APPROVED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Comment by RajaniRanjan** [comment]
LGTM

**Pull request merged**